Fix: exact match should not stem

kdutia opened the pull request:
# Description

Exact match now doesn't perform stemming. 

Added a failing test for 'biodiversity' then fixed it.

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [x] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail (integrated soon)
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [x] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Before submitting

<!-- Please complete this checklist BEFORE submitting your PR to speed along the review process. -->
- [ ] I've read and followed all steps in the [Making a pull request](https://github.com/climatepolicyradar/cpr-sdk/blob/main/.github/CONTRIBUTING.md#making-a-pull-request)
    section of the `CONTRIBUTING` docs.
- [ ] I've updated or added any relevant docstrings following the syntax described in the
    [Writing docstrings](https://github.com/climatepolicyradar/cpr-sdk/blob/main/.github/CONTRIBUTING.md#writing-docstrings) section of the `CONTRIBUTING` docs.
- [ ] If this PR fixes a bug, I've added a test that will fail without my fix.
- [ ] If this PR adds a new feature, I've added tests that sufficiently cover my new functionality.

kdutia committed:
failing test for exact search

kdutia committed:
add not_stemmed fields to vespa schema

kdutia committed:
fix exact search

olaughter commented on tests/local_vespa/test_app/schemas/document_passage.sd:
```diff
@@ -1,5 +1,10 @@
 schema document_passage {
 
+    field text_block_not_stemmed type string {
+        indexing: input text_block | summary | index
+        stemming: none
```
Nice seeing the field-from-another-field in action!

olaughter reviewed (approved):
Looks good to me for when we are ready! If we go ahead now the next step would be a nav-infra schema change before deploying this through the backend

jesse-c reviewed (approved):
LGTM

It's true by default, which I guess makes sense, for Vespa's primary use cases.

kdutia committed:
add prod vespa schema changes; change rank-profile for exact search

kdutia committed:
bump version to 1.9.5

kdutia commented:
@olaughter re-requesting review as i've updated the schema in line with the prod changes, and updated exact match search to use the new `exact_not_stemmed` rank-profile: see [commit here](https://github.com/climatepolicyradar/cpr-sdk/pull/145/commits/a83e2058f8d0918bcb2921a6d54d813e0e2fd42a)
